Editor/UI/Drawers/MetadataWrapperDrawer.cs

```diff
@@ -103,19 +107,41 @@ public override GUIContent GetLabel(object value, Type declaredType)
             return valueDrawer.GetLabel(wrapper.Value, wrapper.ValueDeclaredType);
         }
 
-        private Rect DrawReorderable(Rect rect, MetadataWrapper wrapper, Action<object> changeValueCallback, GUIContent label)
+        private GUIStyle GetStyle(bool isProperBar = false)
```

Comment:
Since everything here is private I guess we can ignore it, but I actually have no clue what a "proper bar" is 🤔